### ISSUE TITLE
[Javascript][Accessibility - Focus is lost for one tab stop]

### ISSUE THREAD

**Issue by ananddidwania** [microsoft/AdaptiveCards#5267]
# Platform

- [ ] JavaScript

# Author or host

Host Name : Microsoft Search

# Version of SDK

"adaptivecards": "2.0.0-rc.0"

# Details

Focus is lost for one tab stop after each elements with adaptive card.
![image](https://user-images.githubusercontent.com/44603844/104426311-c94b8080-55a7-11eb-9aca-c43bbc5fa3e1.png)

Most probably the focus goes to as-designersurface, although focus is not seen.
Another problem is that when focus is lost the Narrator announces "Double click to edit" multiple times which does not makes sense to people accessing with keyboard and narrator.

**Comment by diduanand** [comment]
@matthidinger, @dclaux Can you please look into it.
cc - @atishayv

**Comment by shalinijoshi19** [comment]
@diduanand can you please upgrade to the [latest 2.x](https://www.npmjs.com/package/adaptivecards/v/2.6.0)  version of the JS renderer and see if you are still seeing the issue repro?

**Comment by diduanand** [comment]
@shalinijoshi19 Can you please tell me which version of rendered the https://adaptivecards.io/designer/ is using. Would be great if you can tell how we can also check that.

**Comment by jonmill** [comment]
This issue should be resolved using the latest SDK version, please update the SDK version and re-open if this is still an issue. Thanks!
